src/ptc/util/misc.clj

```diff
@@ -138,7 +138,8 @@
     [bucket (or object "")]))
 
 (defn gcs-object-exists?
-  "Return FILE stat if the gcs object exists."
-  [file]
-  (do-or-nil
-   (shell! "gsutil" "stat" file)))
+  "Return PATH when there is a GCS object at PATH.  Otherwise nil."
+  [path]
+  (when (string? path)
```

Comment:
is `nil` a valid argument?

Reply:
Yes, or at least it is not an invalid one.